examples/http/actor-parking-sensor/README.md

```diff
@@ -40,22 +19,18 @@ docker run --rm -it -d \
 # Install (from the example directory)
 npm install
 
-# Start a RabbitMQ Container (for the binding example part)
-# note: mgmt interface at http://localhost:15672 
-docker run -d --rm --hostname my-rabbitmq --name my-rabbitmq \
-    -e RABBITMQ_DEFAULT_USER=test-user -e RABBITMQ_DEFAULT_PASS=test-password \
-    -p 0.0.0.0:5672:5672 -p 0.0.0.0:15672:15672 \
-    rabbitmq:3-management
-
-# Run the example
+# Build and run the example
+npm run build
 npm run start:dapr
 ```
 
-```
-from(bucket: "my-sensors")
-  |> range(start: v.timeRangeStart, stop: v.timeRangeStop)
-  |> filter(fn: (r) => r["_measurement"] == "sensor-states")
-  |> pivot(columnKey: ["_field"], rowKey: ["_time"], valueColumn: "_value")
-  |> group()
-  |> yield(name: "last")
-```
\ No newline at end of file
+## Visualize
+
+Start grafana on localhost:3001 and use `admin`/`password` to login and visualize the dashboard!
```

Comment:
add link ref to localhost [localhost:3000](localhost:3001)

nit: grafana capitalized (Grafana)